pkg/heatshrink/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=heatshrink
+PKG_URL=https://github.com/atomicobject/heatshrink.git
+PKG_VERSION=7d419e1fa4830d0b919b9b6a91fe2fb786cf3280
+
+.PHONY: all
+
+all: git-download
+	cp Makefile.heatshrink $(PKG_BUILDDIR)/Makefile
```

Comment:
The package's Makefile is mainly meant to build the example CLI application and Linux test cases. The package's docs explicitly advise to just copy the needed files into one's project and add them to the respective build system. So I decided to just copy over the two-line Makefile instead of creating a patch that empties the whole file and then adds the two lines.